ocpmodels/trainers/forces_trainer.py

```diff
@@ -295,11 +296,20 @@ def train(self):
                 if i % self.config["cmd"]["print_every"] == 0:
                     print(self.meter)
 
+                iters += 1
+
+                # Evaluate val set. -1 for after each epoch.
+                eval_every = self.config["optim"].get("eval_every", -1)
+                if iters % eval_every == 0 and iters / eval_every != -1:
```

Comment:
Should this be `if iters % eval_every == 0 and eval_every != -1:`. I didn't quite get what `iters / eval_every` is supposed to capture..

Reply:
Oops sorry, yeah typo.